arm-automation/2015-10-31/swagger/account.json

```diff
@@ -0,0 +1,578 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AutomationManagementClient",
+    "version": "2015-10-31"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
+  ],
+  "produces": [
```

Comment:
Can this be at resource level? We have only one resource that will output test/json